test_runner/fixtures/metrics.py

```diff
@@ -75,9 +75,6 @@ def parse_metrics(text: str, name: str = "") -> Metrics:
 PAGESERVER_PER_TENANT_METRICS: Tuple[str, ...] = (
     "pageserver_current_logical_size",
     "pageserver_resident_physical_size",
-    "pageserver_getpage_get_reconstruct_data_seconds_bucket",
-    "pageserver_getpage_get_reconstruct_data_seconds_count",
-    "pageserver_getpage_get_reconstruct_data_seconds_sum",
```

Comment:
I am confused, it was already a global metric by @skyzh in 2252c5c282.

Reply:
oh it's still at tenant-level not pageserver level.

Reply:
No I misread this. `pageserver_getpage_get_reconstruct_data_seconds_*` is not in globals, apologies for the noise.